examples/conda-forge.yaml

```diff
@@ -91,13 +97,18 @@ python3-netifaces:
 python-paramiko:
   conda-forge: [paramiko]
 sbcl:
```

Comment:
can we do something like?

```
sbcl:
  conda-forge:
    win64: []
    unix: [sbcl]
```

Reply:
We sure can. See https://github.com/ros-forge/vinca/pull/3/commits/a0d5b10c1937eee4fec2579c7c016c98e1223bd8